src/windshaft/filters/collection.js

```diff
@@ -2,44 +2,39 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 
 module.exports = Backbone.Collection.extend({
+
+  /**
+   * @param  {Array.<Filter>} filters Array with the filters to add to the collection.
+   * @param  {Array.<LayerModel>} layers  Array of LayerModel in the same order passed to the WindshaftMap.
+   * @returns {WindshaftFilterCollection}
+   */
   initialize: function (filters, layers) {
     this.layers = layers;
     Backbone.Collection.prototype.initialize.apply(this, filters);
   },
 
   toJSON: function () {
```

Comment:
If there were tests for this and everything was working fine... why do we need to change the tests? :astonished: